airflow/models/dag.py

```diff
@@ -549,6 +550,9 @@ def pickle_id(self):
     def pickle_id(self, value):
         self._pickle_id = value
 
+    def param(self, name, default):
```

Comment:
Can you please add docstring? Pylint will be sad once we start running it over this file :<